### ISSUE TITLE
snips-skill stopping or not auto starting

### ISSUE THREAD

**Issue by ptc** [snipsco/snips-skill-respeaker#26]
I noticed that from time to time, snips-skill-respeaker stops working (no led feedback) or won't autostart after reboot and I have to `sudo systemctl restart snips-skill-server` to get it up again.

**Comment by CoorFun** [comment]
Hello,

If you can reproduce the issue, you can run the command:

```bash
sudo systemtl status snips-skills-server
```

to retrieve the fail log. 

it's easier for me to see the crash reason.

**Comment by ptc** [comment]
No log, just: 
`Unit snips-skills-server.service could not be found.`

**Comment by ptc** [comment]
`Apr 22 15:25:57 snips1 snips-skill-server[549]: WARN:snips_skill_server_lib::handler::nosandbox: child led_animation_1.6 has stopped with status `exit code: 141`
Apr 22 15:25:57 snips1 snips-skill-server[549]: INFO:snips_skill_server_lib::runner            : restarting action led_animation_1.6 (restart_fails: 0)`

**Comment by CoorFun** [comment]
Hello:
Could you please try to manually run `sodo systemctl restart snips-*` ? This is the command that `sam` use to restart  all the snips services.

**Comment by ptc** [comment]
It works then but only until I reboot or it stops working again. Don't know what's going on. Sometimes I can see using `sam watch` that snips is listening bit led won't flash. 
Snips got pretty unstable after the last major update.

**Comment by CoorFun** [comment]
Hello,

The `snips-skill-respeaker` is managed and run by `snips-skill-server`, it stops working after sam update assistant could be caused by skill server is not properly restarted.

SAM will use command
```
sudo systemctl restart snips-*
```
to restart all the snips services. 

But the problem is that this command can be blocked by some error sometimes. 

If you can try to restart the `snips-skill-server` manually when the led is not working, then it should be restored. We will try to fix this issue from sam side.

**Comment by ptc** [comment]
yes whenenver I restart snips-skill-server, it works again. but unfortunately I have to do it evertime I reboot my snips devices.

**Comment by ptc** [comment]
well sometimes it even won't work after I restart skill server.

**Comment by CoorFun** [comment]
When it's not working, you can run command: 
```
journalctl -u snips-skill-server -f
```

This can print the realtime log output of skill server. Let me know the output.

**Comment by ptc** [comment]
```
 -- Logs begin at Thu 2016-11-03 18:16:42 CET. --
Okt 25 11:17:09 snips1 systemd[1]: Started Snips Skill Server.
Okt 25 11:17:10 snips1 snips-skill-server[587]: INFO:snips_skill_server_lib::runner: searching dir for actions: /var/lib/snips/skills/snips-skill-respeaker
Okt 25 11:17:10 snips1 snips-skill-server[587]: INFO:snips_skill_server_lib::runner: found action "/var/lib/snips/skills/snips-skill-respeaker/action-led_animation_1.6.3"
```

So it says it started but LED is not working although snips does what he's asked for.
